tests/auto/jobs/build.py

```diff
@@ -100,49 +111,84 @@ def remove_pr_data(job_obj, pr_repo_loc, repo_dir_str, rt_dir):
 def clone_pr_repo(job_obj, workdir):
     ''' clone the GitHub pull request repo, via command line '''
     logger = logging.getLogger('BUILD/CLONE_PR_REPO')
-    repo_name = 'ufs-community/ufs-srweather-app'
-    branch = 'develop'
-    git_url = f'https://${{ghapitoken}}@github.com/{repo_name}'
-    logger.debug(f'GIT URL: {git_url}')
+
+    app_name = 'ufs-srweather-app'
```

Comment:
Does this need to be hard-coded here? Is it in the config file?

Reply:
I am changing it to break it out from the app address.